server/app/controllers/HomeController.java

```diff
@@ -62,6 +67,12 @@ public CompletionStage<Result> index(Http.Request request) {
                 // If the applicant has not yet set their preferred language, redirect to
                 // the information controller to ask for preferred language.
                 ApplicantData data = applicant.getApplicantData();
+                if (langs.availables().size() <= 1 && !data.hasPreferredLocale()) {
```

Comment:
I think it's okay to leave preferred locale as is for the person. It's highly unlikely that they would be able to set it to some other language (it would have to available at some point, and the removed).